modules/ui/bin/jenkins-build-deploy.sh

```diff
@@ -109,11 +111,12 @@ elif [[ "${version}" == *SNAPSHOT ]]; then
   artifact_repository_id=${nexus_snapshot_repository_id}
 fi
 
-group=`fromPom main ${profile} project.groupId`
+group=`fromPom project.groupId`
 artifact=ui
-path=${nexus_repositories}/${artifact_repository_id}/`echo ${group} | sed "s/\./\//g"`/${artifact}/${version}
-zip=${project}-${artifact}-${profile}-${version}.zip
-zip_path=${path}/${zip}
+group_modified=`echo ${group} | sed "s/\./\//g"`
```

Comment:
@abhishekjain88 same as above: This got removed as I was trying different things to make fromPom work to retrieve project.version - I am going to revert all the changes and just keep the change which made retrieving project.version work and which is adding 'tail -n 1'